### PR TITLE
Add cerebro-npms

### PR DESCRIPTION
**Pull request by FreaKzero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@
 - [cerebro-duck-duck-go](https://www.npmjs.com/package/cerebro-duck-duck-go) - Use Duck Duck Go as your Cerebro App search engine.
 - [cerebro-vaporwave](https://www.npmjs.com/package/cerebro-vaporwave) - Cool text bro but it need more ＶＡＰＯＲＷＡＶＥ.
 - [cerebro-npm](https://www.npmjs.com/package/cerebro-npm) - A Cerebro plugin for searching NPM packages.
+- [cerebro-npms](https://www.npmjs.com/package/cerebro-npms) - Cerebro plugin to search NPM packages via NPMS API
```
Comment (lubien) [inline README.md]: Show have a "." at the end.

Comment (lubien) [inline README.md]: Newly added items should be at the end of it's list.

**Comment by FreaKzero** [comment]
Sorry - my fault

**Comment by lubien** [comment]
Ok. Just one more thing. I need you to squash the commits into one. Forgot to tell you in my last message, sorry.

**Comment by FreaKzero** [comment]
erm - anything else wrong ?

**Comment by lubien** [comment]
Everythings ok now ;)

**Comment by FreaKzero** [comment]
awesome! thank you :)

**Pull request merged**